dotcom-rendering/src/lib/assets.ts

```diff
@@ -84,7 +84,7 @@ const getScripts = (
 
 	return manifestPaths.map((manifestPath) => {
 		const manifest = getManifest(manifestPath);
-		const filename = manifest[file];
+		const filename = manifest[file] ?? '';
```

Comment:
We should probably `throw` or at least record this as an error, because it will lead to errors in production if assets cannot be found!